plugins/modules/ovirt_host_storage_info.py

```diff
@@ -134,26 +142,20 @@ def main():
         # Get Host
         hosts_service = connection.system_service().hosts_service()
         host_id = get_id_by_name(hosts_service, module.params['host'])
-        storage_type = _get_storage_type(module.params)
         host_service = hosts_service.host_service(host_id)
 
-        if storage_type == 'iscsi':
+        if module.params.get('iscsi'):
             # Login
-            iscsi = module.params.get('iscsi')
-            _login(host_service, iscsi)
+            _login(host_service, module.params.get('iscsi'))
 
         # Get LUNs exposed from the specified target
         host_storages = host_service.storage_service().list()
-
-        if storage_type == 'iscsi':
-            filterred_host_storages = [host_storage for host_storage in host_storages
-                                       if host_storage.type == otypes.StorageType.ISCSI]
-            if 'target' in iscsi:
-                filterred_host_storages = [host_storage for host_storage in filterred_host_storages
-                                           if iscsi.get('target') == host_storage.logical_units[0].target]
-        elif storage_type == 'fcp':
-            filterred_host_storages = [host_storage for host_storage in host_storages
-                                       if host_storage.type == otypes.StorageType.FCP]
+        if module.params.get('iscsi') is not None:
```

Comment:
I think "is not None" can be removed both here and on line 157.

Reply:
This is needed for the case when we would want to list all `iscsi` without login.
If I would remove it it would not pass with `iscsi: {}` so it would not filter the StorageType.
The same goes for below.

Reply:
Ack.
You are right :)